tests/add-user-ui.spec.ts

```diff
@@ -1,74 +1,97 @@
+  }
+  return result;
+};
+test.beforeEach(async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+});
+test("Check that page Add User has title", async ({ page }) => {
   await expect(page).toHaveTitle("TS Trainee course");
 });
 
-test("Create button UI", async ({ page }) => {
-  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
-  const el = page.getByRole("button", { name: "Create" });
-  await expect(el).toBeVisible;
-  await expect(el).toHaveClass("btn btn-primary");
-  await expect(el).toHaveCSS("background-color", "rgb(13, 110, 253)");
-  // checking btn reaction to hover
-  await el.hover();
-  await expect(el).toHaveCSS("background-color", "rgb(11, 94, 215)");
+test("Verify Create button design on the Add User page", async ({ page }) => {
+  const createBtn = page.getByRole("button", { name: "Create" });
+  await expect(createBtn).toBeVisible;
+  await expect(createBtn).toHaveCSS("background-color", Colors.lightBlue);
+  await createBtn.hover();
+  await expect(createBtn).toHaveCSS("background-color", Colors.darkBlue);
 });
 
-test("Cancel button UI", async ({ page }) => {
-  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
-  const el = page.getByRole("link", { name: "Cancel" });
+test("Verify Cancel button design on the Add User page", async ({ page }) => {
+  const cancelBtn = page.getByRole("link", { name: "Cancel" });
 
-  await expect(el).toHaveClass("btn btn-secondary");
-  await expect(el).toHaveCSS("background-color", "rgb(108, 117, 125)");
-  // checking btn reaction to hover
-  await el.hover();
-  await expect(el).toHaveCSS("background-color", "rgb(92, 99, 106)");
+  await expect(cancelBtn).toHaveCSS("background-color", Colors.lightGrey);
+  await cancelBtn.hover();
+  await expect(cancelBtn).toHaveCSS("background-color", Colors.darkGrey);
 });
 
-test("User name UI", async ({ page }) => {
-  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
-  const el = page.getByLabel("User Name");
-  const placeholder = await el.getAttribute("placeholder");
+test("Verify User Name field design and error messages on the Add User page", async ({
+  page,
+}) => {
+  const userNameField = page.getByLabel("User Name"); //need to change to XPath
+  const placeholder = await userNameField.getAttribute("placeholder");
 
-  await expect(el).toBeVisible;
+  await expect(userNameField).toBeVisible;
   await expect(placeholder).toEqual("User Name");
-  await expect(el).toHaveValue("");
+  await expect(userNameField).toHaveValue("");
 
   //checking error msg for too short value
-  await el.fill("ab");
-  await expect(el).toHaveValue("ab");
-  const err = page.getByText("Name is too short");
+  let testStr = generateRandomString(2);
+  await userNameField.fill(testStr);
+  await expect(userNameField).toHaveValue(testStr);
+  const err = page.getByText("Name is too short"); //need to change to XPath
```

Comment:
again not so good variable name, it's better to name it like this: 
nameIsTooShortMsg
nameIsTooShortErr